CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+
+Please take a few minutes to read GitHub's guide on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).  It's a quick read, and it's a great way to introduce yourself to how things work behind the scenes in open-source projects.
+
+### Dependencies
+
+If you are introducing a new dependency, please make sure it's added to:
+ * requirements.txt
+
+### Documentation
+
+We are maintaining whole project documentation inside [README.md](/README.md).
+
+#### Changelog
+
+When you are contributing to changelog, please follow these suggestions:
+
+* The changelog is meant to be read by everyone. Imagine that an average user
+  will read it and should understand the changes. `docker_image.mount() via .get_archive()` is
+  not very descriptive. `DockerImage class now utilizes get_archive() from
+  docker-py for its mount() method.` is a more friendly description.
+* Every line should be a complete sentence. Either tell what is the change that the tool is doing or describe it precisely:
+  * Bad: `Use search method in label regex`
+  * Good: `Colin now uses search method when...`
+* And finally, with the changelogs we are essentially selling our projects:
+  think about a situation that you met someone at a conference and you are
+  trying to convince the person to use the project and that the changelog
+  should help with that.
+
+
+### Testing
+
+For testing, we are using [pytest](https://docs.pytest.org/en/latest/) framework. Tests are stored in the [tests](/tests) directory. We recommend to run tests inside docker container using:
+```
+# if TEST_TARGET is empty whole test suite is executed
+make test-in-container TEST_TARGET=<PATH>
+```
+
+Substitute `<PATH>` with path to specific file, for example:
+```
+make test-in-container TEST_TARGET=tests/test_github.py
+```
+
+#### Cleanup after testing
+
+We have an integration test suite in release bot which creates a new project on
+github and tries the functionality in there. It may easily happen that a bunch
+of test repositories will be left out in your account:
+
+ * We have a script which is able to delete all of these, please check
+   `./hack/clean-testing-github-repos`. We suggest reading out the sources
+   first before using it.
+ * The prerequisite for both, the integration tests and the script mentioned
+   above, is to have a token which is able to delete projects.
+
+### How to contribute code to release-bot
+
+1. Create a fork of the `release-bot` repository.
+2. Create a new branch just for the bug/feature you are working on.
+
+   - If you want to work on multiple bugs/features, you can use branches to keep them separate, so that you can submit a separate Pull Request for each one.
+
+3. Once you have completed your work, create a Pull Request, ensuring that it meets the requirements listed below.
+
+### Requirements for Pull Requests
```

Comment:
I'd probably move this and 'How to contribute code to release-bot' before 'Testing'.